client/app/components/gmap.jsx

```diff
@@ -35,7 +35,7 @@ class Gmap extends React.Component {
 
   overlayHeatmap() {
     var heatmapPoints = this.props.heatmapData.map(function(crime) {
-      return new google.maps.LatLng(crime.location.coordinates[0] - 0, crime.location.coordinates[1] - 0);
+      return new google.maps.LatLng(crime.location.coordinates[1] - 0, crime.location.coordinates[0] - 0);
```

Comment:
only thing I'm wondering is why [coords[1], coords[0]] works for you whereas [coords[0], coords[1]] works for me. @tylerarbus @abpeinado which one works for you guys? It could just be me